src/Sarif.FunctionalTests/SarifConverterTestData/AndroidStudio/AndroidStudio_issueLog3_raw.xml.sarif

```diff
@@ -5,21 +5,28 @@
       "toolInfo": {
         "name": "AndroidStudio"
       },
+      "runInfo": {
+        "fileInfo": {
+          "app/src/main/java/microsoft/androidstudioformatexample/MainActivity.java": [
+            {
+              "uri": "app/src/main/java/microsoft/androidstudioformatexample/MainActivity.java",
+              "mimeType": "text/x-java-source"
+            }
+          ]
+        }
+      },
       "results": [
         {
           "ruleId": "'for' loop replaceable with 'foreach'",
           "fullMessage": "<code>for</code> loop replaceable with 'foreach' #loc",
```

Comment:
Look at the implied SARIF issues in this fullMessage output:
1) is it ok to embed formatting hints such as <code> or <bold> in message output? 
2) converter issue: i'm guessing the converter should be removing #loc which I'm guessing is expanded to an actual code location. Just a guess.

I opened #147 to track converter changes. Do we have a sarif item open on #1?